wiki/Tournaments/oPHF/2018/en.md

```diff
@@ -0,0 +1,277 @@
+---
+tags:
+- oPHF 2018
+- oPHF2018
+---
+# osu!Philippines Holiday Friendlies 2018
+
+![oPHF2018 Poster](hf2018poster.png)
+
+> It's now time for some Holiday cheer!
+
+The **osu!Philippines Holiday Friendlies 2018** is a semi-restricted, semi-open Tournament Series hosted by the osu!PH community for the osu!standard, osu!taiko, osu!catch, and the osu!mania game modes. This tournament aims to provide up and coming players an competitive environment where they have a chance at glory themselves.
+
+This serves as the second edition of the tournament.
+
+## Proposed Tournament Schedule
+
+| Event | Timestamp |
+|---:|---|
+| Registration Phase   | 2018-10-13/2018-10-28 |

```

Comment:
Please make the spacing 1 space (Pre and Post content)
Example: | Match Week 1 |